src/app/command-processor.js

```diff
@@ -615,7 +615,7 @@ function createCommand(category, name, description, options){
  */
 function parse(command, args){
 	Yargs.reset();
-	Yargs.wrap(Yargs.terminalWidth());
+	Yargs.wrap(null);
```

Comment:
Is this change on purpose? I don't have strong feelings about wrapping one way or another so you own the final decision. This is the difference for running `particle help` with and without wrapping.

![image](https://user-images.githubusercontent.com/1919681/73183306-971be700-40e8-11ea-9e92-fab89cf8949d.png)

![image](https://user-images.githubusercontent.com/1919681/73183327-9edb8b80-40e8-11ea-84e2-d82a7b0b301a.png)

Reply:
>Is this change on purpose?

yes. i was trying to fix a bug with the param type labels (e.g. `[boolean]', etc) - see the full diff over here 👉 https://github.com/particle-iot/particle-cli/pull/544/commits/bb4936cfc42009f458935f3b5cbb1b5d4a5190dc

>This is the difference for running particle help with and without wrapping.

we're a bit wider than the ideal of 80cols but things should look normal if you expand the terminal window - for that help screen:

```sh
$tput cols
125
```

ultimately, the best thing would be to fix the buggy type label handling without messing with wrapping, etc. i wasn't able to figure out how to thread that needle - all pointers welcome.

in the meantime, i'll highlight this change with the team and get feedback :+1:

Reply:
I think this makes the output consistent for testing, maybe we want to be able to turn this on for testing but not for production.

Reply:
>maybe we want to be able to turn it off for testing

not a bad suggestion :+1:

Reply:
✅ done in https://github.com/particle-iot/particle-cli/pull/544/commits/5f92d8adce427a8a5f3b4d956296dcdef8fbe807